test/parallel/test-fs-open.js

```diff
@@ -3,17 +3,12 @@ const common = require('../common');
 const assert = require('assert');
 const fs = require('fs');
 
-let caughtException = false;
+const expected = new RegExp('^Error: ENOENT: no such file or directory, ' +
+                            'open \'/path/to/file/that/does/not/exist\'$');
 
-try {
-  // should throw ENOENT, not EBADF
-  // see https://github.com/joyent/node/pull/1228
+assert.throws(() => {
   fs.openSync('/path/to/file/that/does/not/exist', 'r');
-} catch (e) {
-  assert.strictEqual(e.code, 'ENOENT');
```

Comment:
Yes exactly, we are now validating the error message and not the `code` property. The test no longer fails if `e.code !== 'ENOENT'`.